models/DataObject/ClassDefinition/Data.php

```diff
@@ -654,6 +654,34 @@ protected function getPreGetValueHookCode(string $key): string
         return $code;
     }
 
+    /**
+     * @return string|null
+     */
+    public function getTypeHintInputType(): ?string {
```

Comment:
I think "parameter type" would be a more appropriate name than "input type".

Reply:
changed it